sdk/python/feast/job.py

```diff
@@ -172,8 +173,11 @@ def to_chunked_dataframe(
             pd.DataFrame:
                 Pandas DataFrame of the feature values.
         """
+
+        # Object is Avro row type object, refer to self.result function for this type
+        records: List[object] = []
```

Comment:
Why are you using object instead of the type?

Reply:
The idea was:
1. The Avro Row will be populated to object thus overriding the type
2. The records don't call any method, it is just being passed around, so it is safe
3. After checking all the tests pass (need to tests for the e2e as well thought)
4. I cannot find what is AvroRow filetype, do you know?
I have tried to dig deeper into the class until I found it was reading only from `args` and `kwargs` without return type, is it byte?

Reply:
Types are there to add safety. It seems like you are looking for a shortcut here.

Reply:
Okay, let me try to simulate this function call to Avro row, and print the type, let me see what is the type.

Although I am certain object should be sufficient, we can see the type to make sure

Reply:
> Although I am certain object should be sufficient
Sufficient to do what?

Reply:
> Sufficient to do what?

O yeah, the object is not sufficient, I thought it will inherit everything, but it doesn't allow you to inherit the function as well.

 I just checked it out, the fastreader.reader will return `List of dict` hence, have rebased the commit. 

Thanks